integration-tests/src/tests/client/features/access_key_nonce_for_implicit_accounts.rs

```diff
@@ -603,7 +601,11 @@ impl ChunkForwardingOptimizationTestData {
         }
     }
 
-    fn process_one_peer_message(&mut self, client_id: usize, requests: NetworkRequests) {
+    fn process_one_peer_message(
+        &mut self,
```

Comment:
Why is the mut necessary here?

Reply:
When a message is received the test environment processes it, which requires a mutable reference to `self`.
For example when a `PartialEncodedChunkRequest` is received, we have to call:
```rust
self.env.process_partial_encoded_chunk_request(
    client_id,
    PeerManagerMessageRequest::NetworkRequests(requests),
);
```